### PR TITLE
Updated a readme for tests

### PR DESCRIPTION
**Pull request by yury-dubinin**
## What is the purpose of the change:

- Updated a readme for tests

### CODE REVIEW

File: packages/web/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# @osmosis-labs/web/e2e
+
+This package contains the Playwright E2E tests and a Keplr wallet
+
+## Environment Variables
+
+By default, configuration is pointing to the [Stage](https://stage.osmosis.zone) environment. Tests will automatically install a wallet. All you need to add is a privat keys for wallets that are being used:
```
Comment (jonator) [inline packages/web/e2e/README.md:7]: typo: `privat`

File: packages/web/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+To install Playwright, please execute `npx playwright install --with-deps chromium` from the /web folder.
+
+To run a Select pair tests, please execute `npx playwright test -g "Test Select Swap Pair feature"` from the /web folder.
+To run a Swap E2E tests, please execute `npx playwright test -g "Test Swap feature"` from the /web folder.
+To run a Monitoring E2E tests, please execute `npx playwright test monitoring --timeout 180000` from the /web folder.
+
+Tests can be executed locally in a browser by changing `headless: true` to `headless: false`.
+
+In CI secrets frontend test privet key is referenced as:
```
Comment (jonator) [inline packages/web/e2e/README.md:23]: typo: `privet`

File: packages/web/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- SOL.wh > 0
+- ETH.axl > 0
+- DAI > 0
+- milkTIA > 0
+- KUJI > 0
+- USDC > 5
+- USDT > 2
+- USDC.eth.axl > 2
+
+Token marked as `> 0` are needed for a portfolio balances test.
```
Comment (jonator) [inline packages/web/e2e/README.md:45]: `Token` > `Tokens`

**Review by jonator — APPROVED** [review]

File: packages/web/e2e/README.md (hunk 1 above)
Comment (yury-dubinin) [reply packages/web/e2e/README.md]: fixed

File: packages/web/e2e/README.md (hunk 3 above)
Comment (yury-dubinin) [reply packages/web/e2e/README.md]: fixed

File: packages/web/e2e/README.md (hunk 2 above)
Comment (yury-dubinin) [reply packages/web/e2e/README.md]: fixed, thx. privet is actually a hi in Russian :)

**Pull request merged**